### PR TITLE
[beez] Can't save content when captcha is enabled

### PR DESCRIPTION
**Pull request by brianteeman**
### Summary of Changes
Add the missing code to display captcha when creating content with the Beez template

### Testing Instructions
Configure and enable the recaptcha plugin
In article options enable captcha on submit
screenshotr14-05-25
Create a menu item type to create an article

### Expected result
Captcha displayed on the front end when creating an article

### Actual result
Test with protostar - captcha displayed
Test with beez - captcha NOT displayed and article cannot be saved

### CODE REVIEW

**Comment by AlexRed** [comment]
I have tested this item :white_check_mark: successfully on 342cf9644cc589a84dce5db6e390e25154bde33a<br /><br />Patch ok for me<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14830">issues.joomla.org/tracker/joomla-cms/14830</a>.</sub>

File: templates/beez3/html/com_content/form/edit.php [hunk 1]
```diff
@@ -74,7 +74,9 @@
 					<?php echo JText::_('JCANCEL') ?>
 				</button>
 			</div>
-
+			<?php if ($this->captchaEnabled) : ?>
+					<?php echo $this->form->renderField('captcha'); ?>
```
Comment (C-Lodder) [inline templates/beez3/html/com_content/form/edit.php]: Only 1 tab required for indentation

**Comment by brianteeman** [comment]
Can we have one more test here please

**Comment by waader** [comment]
I have tested this item :white_check_mark: successfully on 63374b2ce9eaf4393cafa112032a35da522505d4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14830">issues.joomla.org/tracker/joomla-cms/14830</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 63374b2ce9eaf4393cafa112032a35da522505d4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14830">issues.joomla.org/tracker/joomla-cms/14830</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14830">issues.joomla.org/tracker/joomla-cms/14830</a>.</sub>

**Comment by brianteeman** [comment]
thanks

**Pull request merged**